### ISSUE TITLE
Bluetooth Setting On/Off Toggle State should be persistent through Reboots. It currently defaults to On.

### ISSUE THREAD

**Issue by ChaosBlades** [pop-os/pop#1748]
Currently Bluetooth's On/Off toggle will default to On upon reboot. This is a security issue. The only way to keep the Bluetooth state  persistent through reboots is to stop and disable the service.

https://www.reddit.com/r/pop_os/comments/o8350v/stop_bluetooth/

**Comment by ClosetCoder** [comment]
I agree completely. The only work around I know of currently (since I don't use blue-tooth on my laptop), is;
'sudo systemctl disable bluetooth.service'
But using this option makes the menu show as it does in the attached image.
![Screenshot from 2021-06-30 22-01-24](https://user-images.githubusercontent.com/14336939/124370016-1ccd8b80-dc27-11eb-937f-7a0383f165bb.png)
The blue-tooth state is off, yet it still provides the option to turn it off?

**Comment by crawfxrd** [comment]
> The blue-tooth state is off, yet it still provides the option to turn it off?

Bluetooth service is stopped, but the device is still enabled. Clicking it, or blocking it via rfkill, will cause it to go away.

```
rfkill block bluetooth
```
